rb/spec/integration/selenium/webdriver/spec_support/test_environment.rb

```diff
@@ -152,6 +158,11 @@ def create_driver!(listener: nil, **opts, &block)
         private
 
         def build_options(args: [], **opts)
+          browser = self.browser
+          if browser.to_s == 'Safari Technology Preview'
```

Comment:
Is driver not going to be `safari_preview` here? If so it should be set properly in `#safari_preview_options` called from line 168. Also, we really should move this logic to `SafariOptions` class like Java has it. Setting it on a class method seems kind of weird at this point.

Reply:
We read the browser name from the ENV variable, and it's expected to be `Safari Technology Preview`. I can use something else, but that's what https://github.com/SeleniumHQ/selenium/blob/ci-merge/rb/lib/selenium/webdriver/safari/options.rb expects.

Reply:
Right, but we also have `WD_SPEC_DRIVER` which should already be toggling this.

Reply:
You are right, it's not needed anymore.

Reply:
It works a bit weird when used with `remote`, but I'd like to later make a separate configuration for a channel, something like this:

```
bazel test --define browser=safari --define channel=preview rb/spec/...
bazel test --define browser=chrome --define channel=beta rb/spec/...
bazel test --define browser=firefox --define channel=nightly rb/spec/...
```

Reply:
that would require pinning browsers?

Reply:
Not necessarily, I need to try out first.

Reply:
what is weird with remote? isn't it just:
```
bazel test --define browser=safari-preview --define remote=true
```
Or do you mean the implementation where we're setting the value on the class?